Sming/Core/Data/Stream/JsonObjectStream.h

```diff
@@ -24,7 +24,7 @@ class JsonObjectStream : public MemoryDataStream
 public:
 	/** @brief  Create a JSON object stream
     */
-	JsonObjectStream() : rootNode(buffer.createObject())
+	JsonObjectStream() : doc(1024)
```

Comment:
Given that buffer size is fixed, I'd add a parameter here `JsonObjectStream(size_t capacity = 1024) : doc(capacity)`